setup.py

```diff
@@ -18,7 +18,7 @@
     'Programming Language :: Python :: 3.5',
     'Topic :: Communications'
 ]
-INSTALL_REQUIRES = ['requests']
+INSTALL_REQUIRES = ['requests', 'click']
 EXTRAS_REQUIRE = {}
```

Comment:
We might want to make the CLI an "extra"? Not completely sure how I feel about that, what do you think?